app/projects/views.py

```diff
@@ -599,7 +601,7 @@ def project_details(request, project_key):
                "user_jwt": user_jwt,
                "user_access_request": user_access_request}
 
-    if not access_granted:
+    if not access_granted or not project.is_contest:
```

Comment:
What would a user see when they _are_ granted access to a non-contest project?

Reply:
Nothing yet. The Berson project will be handled by a human to grant access for now.